packages/api/package.json

```diff
@@ -17,7 +17,7 @@
     "serve": "LOCALSTACK_HOST=localhost NODE_ENV=test FAKE_AUTH=true cumulus-api serve",
     "serve-oauth": "LOCALSTACK_HOST=localhost NODE_ENV=test cumulus-api serve",
     "serve-remote": "cumulus-api serve",
-    "prepublishOnly": "PRODUCTION=true npm run build"
+    "prepare": "npm run build"
```

Comment:
Did we change whatever was implementing the `prepublishOnly` script to use `prepare` instead? Or was nothing using `prepublishOnly`?

Reply:
`prepublish` has been deprecated: https://github.com/lerna/lerna/tree/master/commands/bootstrap#--ignore-prepublish